near-sdk/src/lib.rs

```diff
@@ -9,6 +9,9 @@ pub use near_sdk_macros::{
 #[cfg(feature = "unstable")]
 pub mod store;
 
+#[cfg(feature = "unstable")]
+pub use environment::hash;
```

Comment:
hmm yeah I did want it module scoped because I didn't want these things to be accidentally used. Possibly I need to think this through more before before this comes in or definitely before becoming stable as to how this will fit in